src/Tools/dotnet-getdocument/src/ProjectOptions.cs

```diff
@@ -27,6 +27,12 @@ public void Configure(CommandLineApplication command)
         Platform = command.Option("--platform <Target>", Resources.PlatformDescription);
         ProjectName = command.Option("--project <Name>", Resources.ProjectDescription);
         RuntimeFrameworkVersion = command.Option("--runtime <RUNTIME_IDENTIFIER>", Resources.RuntimeDescription);
+        command.Option("--prefix-output", Resources.PrefixDescription);
```

Comment:
Why do we need this changes? I believe the options in `GetDocumentCommand` is sufficient for this case.